src/index.js

```diff
@@ -1,7 +1,48 @@
+const client = require('prom-client');
 const config = require('config');
 const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
+const Storage = require('./statuscakedata');
 
-this.interval = setInterval(fetchStatuCakeData, config.statuscake.interval);
+const regex = /\s?([-])\s?/;
+
+const getTagsFromTitle = Title => Title.split(regex);
+
+const g = new client.Gauge({
+  name: 'metric_loadtime',
+  help: 'metric_help',
+  labelNames: ['customer', 'service', 'device', 'version'],
+});
+const h = new client.Gauge({
+  name: 'metric_filesize',
```

Comment:
`frontend_pagespeed_filesize_bytes` and it should be converted in bytes 👯

Reply:
Thanks.